vast/inline.go

```diff
@@ -28,14 +28,7 @@ func (inline *InLine) Validate() error {
 		errors = append(errors, ErrInlineMissCreatives)
 	}
 
-	if inline.Pricing != nil {
-		if err := inline.Pricing.Validate(); err != nil {
-			ve, ok := err.(ValidationError)
-			if ok {
-				errors = append(errors, ve.Errs...)
-			}
-		}
-	}
+	// No necessary to validate pricing which is supported by VAST 3.0.
```

Comment:
actually the price validate did nothing, do we still need to remove it?

Reply:
Yes, I'd like to remove it since it is VAST 3.0